test/functional/Java8andUp/src_110_up/org/openj9/test/annotation/ContainsRuntimeAnnotationTest.java

```diff
@@ -72,6 +73,16 @@
 	@MyFieldAnnotation4
 	int notStable_MultipleAnnotations = 0;
 
+	@Ann(a=1, b='a', c="abcd",
+		d={"aaa", "bbb"},
+		e=En.b,
+		f=@Ann2(a=1, b="abcd", c=@Ann3(a=Object.class)),
+		g={@Ann2(a=1, b="abcd", c=@Ann3(a=Object.class)), @Ann2(a=1, b="abcd", c=@Ann3(a=Object.class))})
+	@Single(a="first")
+	@Single(a="second")
+	@Stable
```

Comment:
for this one, I wouldnt add `@Stable` unsless you know it will be searched last. If not, best to omit it, that way you know each of the other annotations will be searched